clients/src/test/java/org/apache/kafka/clients/MetadataTest.java

```diff
@@ -199,6 +199,35 @@ public void onMetadataUpdate(Cluster cluster) {
             new HashSet<>(Arrays.asList("topic", "topic1")), topics);
     }
 
+    @Test
+    public void testTopicExpiry() throws Exception {
+        // Test that topic is expired if not used within a refresh interval
+        long time = 0;
```

Comment:
Since `0` is a sentinel value, this is probably not a safe value to use for `time` in this test. It does work in this case because the time is only used in that method by adding the `metadataExpireMs`, but this seems risky for catching errors introduced by future changes.